Update active-directory-b2c-reference-oauth-code.md

ArmanvL opened the pull request:
When refreshing a token it required me to provide both a client_id and a client_secret in order to properly execute the refresh token flow. The documentation however only mentioned the client_id. Performing the request as described in the current iteration of the documentation caused a 400 Bad request with a message requesting a client_secret to be provided.

After adding the client_secret to the requests data it worked and I was able to successfully refresh the token.

I added the client_secret to the documentation to make sure others can avoid this small issue.

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ArmanvL committed:
Update active-directory-b2c-reference-oauth-code.md

When refreshing a token it required me to provide both a client_id and a client_secret in order to properly execute the refresh token flow. The documentation however only mentioned the client_id. Performing the request as described in the current iteration of the documentation caused a 400 Bad request with a message requesting a client_secret to be provided.

After adding the client_secret to the requests data it worked and I was able to successfully refresh the token.

I added the client_secret to the documentation to make sure others can avoid this small issue.
